index.html

```diff
@@ -596,6 +596,43 @@ <h2>On behalf of organisations</h2>
+				
+				<p>Silniki wyszukiwania to potężne narzędzia, dzięki którym można rozwiązać ten problem. Jednakże Google zdecydowało stworzyć specjalną pozycję na samej górze wyników wyszukiwania (dla artykułów) oraz dodaje specjalną ikonę <em>błyskawicy</em> (dla pozostałych treści), ale tylko dla wydawców korzystających z technologii kontrolowanej przez Google, udostępnianej przez Google w ramach ich infrastruktury, w ramach adresu URL Google oraz wewnątrz interfejsu stworzonego przez Google. (<a href='https://developers.google.com/search/docs/guides/about-amp'>źródło</a>)</p>
+				
+				<p>Idea AMP nie jest sama w sobie problematyczna, ale dwa aspekty jej implementacji umacniają Google na pozycji <em>de facto</em> standardowej platformy udostępniającej treść w sieci internetowej, a sam Google próbuje zwiększyć użycie AMP:</p>
+				
+				<ol>
+					<li>Treść, która dostępna jest wewnątrz AMP (a tym samym wewnątrz domeny Google), otrzymuje dodatkową promocję w ramach wyników wyszukiwania, w tym: najwyższą pozycję dla artykułów informacyjnych.</li>
+					<li>Kiedy użytkownik decyduje się na odwiedzenie witryny z treścią promowaną przez Google, bezwolnie pozostaje wewnątrz ekosystemu Google.</li>
+
+				</ol>
+				
+				<p>Jeżeli poprawa wygody użytkowników sieci internetowej jest prawdziwym celem AMP, pragniemy zaproponować kilka prostych zmian, które na to pozwolą, a jednocześnie nie zagrożą dynamice i konkurencyjności sieci internetowej i pozostawią ją skierowaną przede wszystkim na potrzeby konsumenta:</p>
+				
+				<ol>
+					<li>Zamiast promować w ramach wyników wyszukiwania w sposób szczególny witryny internetowe korzystające z AMP obdarzcie profitami wszystkie strony, które spełniają dane kryterium, obiektywne i neutralne kryterium wydajności takie jak <a href='https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/metrics/speed-index'>Speed Index</a>. Wówczas wydawcy treści będą mogli skorzystać z dowolnie wybranego przez nich rozwiązania technicznego.</li>
+					<li>Nie wyświetlajcie treści innych podmiotów w ramach swojej własnej witryny, jeżeli fakt używania produktu Google nie jest oczywisty dla użytkownika. Google może oczywiście uruchomić <em>czytnik aktualności</em>, ale niedopuszczalne jest wyświetlanie witryn z markami innych podmiotów w ramach adresu URL Google, niedopuszczalne jest także wymaganie od innych podmiotów, aby używały hostingu Google w celu pojawienia się w ramach wyników wyszukiwania.</li>
+				</ol>
+				
+				<p>Nie chcemy zatrzymywać dalszego rozwoju AMP, a proponowane przez nas zmiany tego nie wymagają. Pochwalamy także silniki wyszukiwania, które premiują witryny internetowe zoptymalizowane do szybkiego wyświetlania. AMP może pozostać jedną z wielu technologii, które pozwolą wydawcom treści tworzyć szybsze strony internetowe i spełniać oczekiwania swoich użytkowników.</p>
```

Comment:
Once more: I'd replace "silniki wyszukiwania" with "wyszukiwarki".

Reply:
That's a minor change, but yeah, why not.